hack/kubestellar.sh

```diff
@@ -0,0 +1,219 @@
+#!/usr/bin/env bash
+# Copyright 2023 The KCP Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Usage: $0 (start or stop | -v)
+
+# Purpose: deploy the kubestellar platform. The following components are created:
+#           (a) 1 kcp workspace: edge service provider workspace (espw)
+#           (b) 3 kubestellar controllers: edge-scheduler, mailbox-controller and placement-translator
+
+# Assumption: kcp server is running.
```

Comment:
I think it would be clearer to say that "kcp server is running" is a _requirement_.  It is checked in this script.